src/test/java/expediatestrunner/FilterTest.java

```diff
@@ -0,0 +1,30 @@
+package expediatestrunner;
+
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+import expediaproductobjects.FilterOptions;
+import expediaproductobjects.LandingPage;
+import expediaproductobjects.SearchBar;
+
+public class FilterTest extends Hooks {
+	
+	//this test searches for hotels with the given location and dates, and filters the results based on desired star rating and price range
+	@Test(enabled=true)
+	public void test() throws InterruptedException {
+		
+		SearchBar searchBar = new SearchBar(driver);
+		LandingPage landing = new LandingPage(driver);
+		FilterOptions filterOptions = new FilterOptions(driver);
+		
+		if(driver.findElement(By.xpath(landing.overlayCloseButton)).isDisplayed()) {
+			searchBar.click(landing.overlayCloseButton);
+		}
+		
+		searchBar.singleDestinationQuery("Boston, Massachusetts United States", "Wed Dec 01 2021", "Sat Dec 04 2021");
+		Thread.sleep(5000);
+		filterOptions.starRating(4);			//enter desired star rating for hotel
```

Comment:
Did you try implementing starRating() to take n parameters? n being the star range (3, 4, and 5)

Reply:
Yes starRating() takes in an int variable for the amount of stars you want and filters it.
On the hotels.com website you have to click each star rating individually. For example if you want 3,4 and 5 stars you have to click 3 4 and 5 separately. So you would need to run the method 3 times (starRaing(3), starRating(4), starRating(5)). I ran this already it works